008-SIP2-checkout_bugfix.patch

Of some unknown reason, the Circulation.pm module croaks when using Reserves.pm import
MoveReserve in SIP2 checkout. This patch should be unneccesary, and could hopefully
be removed some day.

diff --git a/C4/Circulation.pm b/C4/Circulation.pm
index 4d0e7d5..94bf62c 100644
--- a/C4/Circulation.pm
+++ b/C4/Circulation.pm
@@ -1316,7 +1316,7 @@ sub AddIssue {
                 AddReturn( $item->{'barcode'}, C4::Context->userenv->{'branch'} );
             }
 
-            MoveReserve( $item->{'itemnumber'}, $borrower->{'borrowernumber'}, $cancelreserve );
+            C4::Reserves::MoveReserve( $item->{'itemnumber'}, $borrower->{'borrowernumber'}, $cancelreserve );
 
             # Starting process for transfer job (checking transfert and validate it if we have one)
             my ($datesent) = GetTransfers( $item->{'itemnumber'} );